core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2063,6 +2063,7 @@ class SparkContext(config: SparkConf) extends Logging {
    * @param jobId the job ID to cancel
    * @throws InterruptedException if the cancel message cannot be sent
```

Comment:
It seems fine

- Scala

  <img width="780" alt="2016-11-26 1 05 57" src="https://cloud.githubusercontent.com/assets/6477701/20637897/1a78be2a-b3d9-11e6-939a-47c202a50037.png">

- Java

  <img width="570" alt="2016-11-26 1 05 52" src="https://cloud.githubusercontent.com/assets/6477701/20637898/1eded54e-b3d9-11e6-90a5-5b9c34ec0831.png">